exercises/bank-account/uBankAccountTests.pas

```diff
@@ -37,16 +37,19 @@ implementation
 
 procedure BankAccountTests.Returns_empty_balance_after_opening;
 var account: IBankAccount;
+    Expected: double;
 begin
+  Expected := 0.00;
```

Comment:
The tests should probably speak more to the programming technicalities.  So the numbers being `10.0` and `0.0` is more technically accurate (and as concise as needed), otherwise it conveys a potential that the trialing 0's have some significance.

Also consider if the test promotes discussion and choices.